src/components/MenuBar.vue

```diff
@@ -34,6 +41,41 @@
 							:aria-label="t('text', 'Insert emoji')"
 							:aria-haspopup="true" />
 					</EmojiPicker>
+					<Actions v-else-if="icon.class === 'icon-image'"
```

Comment:
I'll have a look.

Reply:
It's fine on my side with NC 24 :thinking: 
![col](https://user-images.githubusercontent.com/11291457/142861534-e52c0a2e-73a8-4914-bbec-a1f5df436bbb.png)

Reply:
Interesting. I'm on latest NC 24 too :thinking: Will have to further look into it then.

Reply:
I took another look and it seems like `vertical-align: top` is missing. The other action icons have it either through `menubar.button`, `menubar.submenu` or `menubar.menuitem-emoji` styling.

```suggestion
					<Actions v-else-if="icon.class === 'icon-image'"
						class="submenu"
```

Reply:
Oh I realized there actually is an alignment issue on my side too but it's the 3 dots menu icon rather than the image one. Anyway, using the submenu is fine. Done, pushed. Thanks!